src/commcare_cloud/commands/ansible/helpers.py

```diff
@@ -30,6 +30,7 @@ def __init__(self, args, environment=None):
             assert args.env_name == environment.name, (args.env_name, environment.name)
         self.environment = environment
         self._stdout_callback = getattr(args, 'stdout_callback', None)
+        self.ansible_logfile = ansible_logfile
```

Comment:
This allows us to redirect the ansible output to a specific file (other than ansible.log)